include/braincloud/BrainCloudRTT.h

```diff
@@ -17,6 +17,14 @@ namespace BrainCloud
 	public:
 		BrainCloudRTT(RTTComms* in_comms, BrainCloudClient* in_client);
 		
+		enum RTTConnectionStatus
+        {
```

Comment:
unimportant: try copy paste the tabs from previous line to align those.
One day we should remove all tabs...

Reply:
ill fix that right now whoops